server/test/testData.js

```diff
@@ -0,0 +1,50 @@
+const testData = {
```

Comment:
we don't need to test routes on fake data,
we need to build fake data first as we did before
so afterAll tests run build 
and after all test close connection